src/server-extension/resolvers/top_collections.ts

```diff
@@ -67,6 +68,9 @@ export class CollectionRow {
     @Field({ nullable: true })
     verifiedAt!: string
 
+    @Field({ nullable: false })
+    createdAt!: string
```

Comment:
```suggestion
    created_at!: string
```
So here as well, right?

Reply:
Yes here as well, for both created_at and verified_at